Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,26 @@
--  user wants to see the info page 
--  user wants to apply as artist 
-- user wants to donate 
--  user wants to have a prostictic worked on
\ No newline at end of file
+Actors: Visitor, Artist, Admin
+
+Visitor - View info page, View Gallery, Donate, RequestProstetic, Liking a Picture.
+A visitor would use this website for geting information or donate as amputee a prostetic or donate as a company.Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- can also request that his/her prostetic is send to artist to make some art of it.
+- to donate a protestic he/she fill in a form and the admin of the site would contact the visitor. 
```

Comment:
**Donate prosthetic**- how to get to the donation page? What info is needed? Prosthetic is not the only thing that can be donated. Money as well. Or attention, advertisements. We need to think about it.
Admin part is not relevant.